modules/tensor_mechanics/doc/content/documentation/systems/Materials/ComputeVariableBaseEigenStrain.md

```diff
@@ -1,14 +1,38 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+# Compute Variable Base EigenStrain
 
-# ComputeVariableBaseEigenStrain
+!syntax description /Materials/ComputeVariableBaseEigenStrain
 
-!alert construction title=Undocumented Class
-The ComputeVariableBaseEigenStrain has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
+## Description
 
-!syntax description /Materials/ComputeVariableBaseEigenStrain
+The material `ComputeVariableBaseEigenStrain` calculates a Rank-2 tensor eigenstrain
+as a function of a Rank-2 tensor base and a scalar material property.
+\begin{equation}
+  \label{eqn:variable_base_eigenstrain}
+  \mathbf{\epsilon} = p \mathbf{T} + \mathbf{A}
+\end{equation}
```

Comment:
Describe what \epsilon is as well, (and maybe it should be called something like $\boldsymbol{\epsilon}_{eigen} to indicate that it's the eigenstrain, rather than the total strain).